src/charm.py

```diff
@@ -516,7 +522,7 @@ def bootstrap_cluster(self, event: ops.framework.EventBase) -> None:
 
             if error_socket_not_exists in e.stderr:
                 event.defer()
-                return
```

Comment:
Suggest to add info logging here, and debug log at the method entry

Reply:
Similarly, this will generate a log by sunbeam guard exception handling.